src/huggingface_hub/__init__.py

```diff
@@ -52,6 +52,9 @@
 # WARNING: any comment added in this dictionary definition will be lost when
 # re-generating the file !
 _SUBMOD_ATTRS = {
+    "_activity_api": [
+        "UserLikes",
+    ],
```

Comment:
not sure about `activity_api`, cc @LysandreJik

Reply:
yep, trying to move data structures out of hf_api (that is really getting close to 4k LOC now with a lot of stuff not directly related to the Client itself). But in this case I agree it's quite empty

Reply:
> Would it make sense to move the existing dataclasses to a specific module instead? (things like model info, commit info, etc)

yep ok with this. I'll do it in a separate PR

Reply:
For now I just removed it and put the dataclass in `hf_api.py`

Reply:
good call, that was it stays flexible for the future!